src/__tests__/actions/resetPasswordActions.test.js

```diff
@@ -0,0 +1,209 @@
+    const newPassword = {
+      password: '@Qwr121324d',
+      newPassword: '@Qwr121324d',
+    };
+    await store.dispatch(resetPasswordAction({ token, newPassword }))
+      .then(async () => {
+        const calledActions = store.getActions();
+        expect(calledActions).toEqual(expectedActions);
+      });
+  });
+  it('it Should dispatch password reset successfully', async () => {
+    moxios.wait(() => {
+      const request = moxios.requests.mostRecent();
+      request.respondWith({
+        status: 200,
+        response: {
+          message: 'Password reset successfull!',
+          data: '$2b$10$zkLNRjMo16sAHEVMcAe9uOSx/zCiATaBaLJWy3e2OPtZiobn5TULS',
+        },
+      });
+    });
+
+    const expectedActions = [
+      {
+        spinner: false,
+        type: SPINNER_STATUS,
+      }, {
+
+        type: RESET_PASSWORD_SUCCESS,
+        payload: {
+          message: 'Password reset successfull!',
+          data: '$2b$10$zkLNRjMo16sAHEVMcAe9uOSx/zCiATaBaLJWy3e2OPtZiobn5TULS',
+        },
+      },
+      {
+        type: ERROR,
+        error: {},
+        status: 'success',
+      },
+    ];
+    store = mockStore({});
+    const token = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6OCwiZmlyc3ROYW1lIjoiVmljdG9yIiwibGFzdE5hbWUiOiJLYXJhbmd3YSIsImVtYWlsIjoidmljdG9ya2FyYW5nd2E0QGdtYWlsLmNvbSIsInBhc3N3b3JkIjoiJDJiJDEwJDVFbDA1TUF0MGY1Z0V4Tlgwa1Q3VnVrVWhudG9RQjVoT0xKdXRPRklKLmxvY2YyaGFib0RhIiwiaXNWZXJpZmllZCI6dHJ1ZSwic2lnbnVwVHlwZSI6ImZhY2Vib29rIiwicm9sZUlkIjozLCJjcmVhdGVkQXQiOiIyMDE5LTEyLTMwVDEwOjE1OjQyLjYyNloiLCJ1cGRhdGVkQXQiOiIyMDIwLTAxLTAzVDEzOjU4OjI3LjUwNVoiLCJpYXQiOjE1NzgwNjI2Nzl9.E2fDvfHp38bvmw1DXFwD0fTIjRa-igYWq8mhc86DWEg';
+    const newPassword = {
+      password: '@Qwr121324d',
+      newPassword: '@Qwr121324d',
+    };
+    await store.dispatch(resetPasswordAction({ token, newPassword }))
+      .then(async () => {
+        const calledActions = store.getActions();
+        expect(calledActions).toEqual(expectedActions);
+      });
+  });
+  it('it Should update email input with actions', async () => {
+    const expectedActions = {
+      type: PASSWORD_RESET_EMAIL,
+      payload: 'test@user.com',
+    };
+    const email = 'test@user.com';
+
+
+    const calledActions = updateEmailInput(email);
+    expect(calledActions).toEqual(expectedActions);
+  });
+  it('it Should update password input with actions', async () => {
+    const password = {
+      password: 'yes@Lord1',
+      confirmPassword: 'yes@Lord1',
+    };
+    const expectedActions = {
+      type: RESET_PASSWORD_INPUT,
+      payload: password,
+    };
+    const calledActions = updatePasswordInput(password);
+    expect(calledActions).toEqual(expectedActions);
+  });
+  it('it Should update spinner status with actions', async () => {
+    const expectedActions = {
+      type: SPINNER_STATUS,
+      spinner: true,
+    };
+    const calledActions = updateSpinnerStatus(true);
+    expect(calledActions).toEqual(expectedActions);
+  });
+});
```

Comment:
I recommend you move mock data into their own file so that another person can re-use it.